test/truffle/compiler/pe/core/eval_pe.rb

```diff
@@ -15,3 +15,7 @@
 example "eval([1, 2, 3].inspect)[1]", 2
 
 tagged counter example "eval(rand.to_s)"
+
+example "eval('14', binding)", 14
+
+example "b = binding; eval('temp = 14', b); b = eval('temp', b)", 14
```

Comment:
Fixed. I'd originally had it to ensure the binding couldn't escape the expression, but that doesn't cause an issue here.